### ISSUE TITLE
A way to enable HEN without loading .sprx 

### ISSUE THREAD

**Issue by brandflake11** [PS3Xploit/PS3HEN#27]
Hello all,

I have recently installed ps3xpad on a super slim ps3 with ps3hen. However, this is now stopping everything from loading when ps3hen is loaded, including the ftp server function, making the system pretty much unusable when loading ps3hen. 

Is there a way to enable HEN without loading the .sprx plugins from cobra in the boot_* file on the hdd0? Like, if I hold a button and enable HEN, will it enable HEN without loading the .sprx plugins?

Thank you for your time and help with this matter.

**Comment by ranjithran** [comment]
Just reinstall fw in recovery mode

**Comment by brandflake11** [comment]
Thanks for the reply, I actually discovered that if you plug in usb drive with no plugins text document, or a blank plugins text document (called boot_*, can't remember the name) then no plugins will load when enabling HEN. This fixed my issue, but it would be a great feature to hold a button down when enabling HEN to not load plugins, so that one did not have to do this workaround, especially when you are new to this. I'll close this though, since my issue has been solved.
